### PR TITLE
sci-mathematics/drgeo: EAPI bump 4 -> 6

### PR DESCRIPTION
**Pull request by g-braeunlich**

### CODE REVIEW

File: sci-mathematics/drgeo/drgeo-1.1.0-r1.ebuild [hunk 1]
```diff
@@ -18,39 +16,35 @@ SRC_URI="
 
 SLOT="0"
 LICENSE="GPL-2"
-KEYWORDS="amd64 ~ppc x86"
+KEYWORDS="~amd64 ~ppc ~x86"
 IUSE="nls"
 
 RDEPEND="
-	x11-libs/gtk+:2
+	x11-libs/gtk+:2=
```
Comment (SoapGentoo) [inline sci-mathematics/drgeo/drgeo-1.1.0-r1.ebuild]: you sure about this? GTK+ is usually ABI stable for major SLOTs. And if it wasn't, the GNOME guys will likely not introduce subslots for GTK, so it's pointless either way.

File: sci-mathematics/drgeo/drgeo-1.1.0-r1.ebuild [hunk 2]
```diff
@@ -18,39 +16,35 @@ SRC_URI="
...
-	if use nls; then
-		cd "${WORKDIR}"/${DOC}
-	else
-		cd "${WORKDIR}"/${DOC}/c
-	fi
-	emake install DESTDIR="${D}"
+	emake -C "${WORKDIR}"/${DOC}/$(usex nls "" c) install DESTDIR="${D}"
+	doicon glade/${PN}.png
 }
+	# Fix desktop file
```
Comment (SoapGentoo) [inline sci-mathematics/drgeo/drgeo-1.1.0-r1.ebuild]: TODO? :wink:

File: sci-mathematics/drgeo/drgeo-1.1.0-r1.ebuild [hunk 3]
```diff
@@ -18,39 +16,35 @@ SRC_URI="
...
 	# tarball and so can be installed. Just create the make install target.
-	cd "${WORKDIR}"/${DOC}
+	cd "${WORKDIR}"/${DOC} || die
 	econf
 }
 
 src_install() {
+	sed -i -e "s/gnome-drgenius.png/${PN}/" \
+		-e '/^Categories=/s/Application;//' \
+		${PN}.desktop || die
```
Comment (SoapGentoo) [inline sci-mathematics/drgeo/drgeo-1.1.0-r1.ebuild]: I'd move this to `src_prepare`

Comment (g-braeunlich) [inline sci-mathematics/drgeo/drgeo-1.1.0-r1.ebuild]: Does not work. The desktop file does not exist at the stage of src_prepare

Comment (SoapGentoo) [inline sci-mathematics/drgeo/drgeo-1.1.0-r1.ebuild]: alright

**Pull request closed without merging**